docroot/themes/humsci/humsci_basic/src/scss/_base.scss

```diff
@@ -1,8 +1,18 @@
 @charset "UTF-8";
 
-// Decanter's use of global variables won't allow us to import with @use
-// @use 'decanter/core/src/scss/components/brand-bar/index';
-@import 'decanter/core/src/scss/decanter-no-markup';
+@import
+  'decanter/core/src/scss/decanter-no-markup',
```

Comment:
note to myself: no-markup brings in everything in the utilities folder, like variables, mixins, functions, and placeholders that can be extended